deployment/helm/node-feature-discovery/templates/worker.yaml

```
@@ -122,6 +122,9 @@ spec:
         - name: host-lib
           hostPath:
             path: "/lib"
+        - name: host-proc-swap
```

Suggest a small change to naming:

```suggestion
        - name: host-proc-swaps
```

You also need to add this in volumeMounts of the container.

---

updated. both name and added it to volumeMounts.